guides/developer/computeinstance-catalogitem-guide.md

```
@@ -0,0 +1,488 @@
+  - [Server-side processing](#server-side-processing)
+- [User operations](#user-operations)
+  - [Create a VM from a catalog item](#create-a-vm-from-a-catalog-item)
+- [Troubleshooting](#troubleshooting)
+
+---
+
+## Overview
+
+OSAC uses a three-level hierarchy to provision compute instances:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)